package.json

```diff
@@ -105,8 +105,11 @@
 	},
 	"trustedDependencies": [
 		"@biomejs/biome",
+		"@swc/core",
```

Comment:
I'm not sure where these come from, but they aren't in the main branch - I'll work on removing them

Reply:
Looks proper now =)